src/components/learner-credit-management/data/tests/constants.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
[curious/clarification] Is this ESLint disable rule still relevant and/or could it be removed?

Reply:
I guess it got autogenerated from my IDEs auto-linting feature. Removed, and the warning doesn't seem to be replicable 🤷🏽